pom.xml

```diff
@@ -51,7 +51,7 @@
         <dep.jdbi3.version>3.4.0</dep.jdbi3.version>
         <dep.drift.version>1.14</dep.drift.version>
         <dep.tempto.version>174</dep.tempto.version>
-        <dep.gcs.version>2.0.0</dep.gcs.version>
+        <dep.gcs.version>2.1.1</dep.gcs.version>
```

Comment:
2.1.3 is the latest, should we use that instead?

Reply:
Yep, thanks!